lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -70,9 +70,20 @@ def void(authorization, options={})
         commit('cancel', post)
       end
 
+      def store(credit_card, options={})
+        requires!(options, :order_id)
+        post = init_post(options)
+        add_invoice(post, 0, options)
+        add_payment(post, credit_card)
+        add_extra_data(post, options)
+        add_recurring(post, options)
+        add_address(post, options)
+        commit('authorise', post)
+      end
+
       def verify(credit_card, options={})
         MultiResponse.run(:use_first_response) do |r|
-          r.process { authorize(100, credit_card, options) }
+          r.process { authorize(0, credit_card, options) }
```

Comment:
I'm guessing that this change is intentional. $0 auths are supported now?

Reply:
I assumed so from the commit message

Reply:
🙈 I didn't read the message. Whoops.